src/data/data.js

```diff
@@ -1,5 +1,10 @@
 //>>excludeStart("exclude", pragmas.exclude);
-define([ "shoestring" ], function(){
+define([
+	"shoestring",
+//>>includeStart("development", pragmas.development);
+	"dom/is"
+//>>includeEnd("development");
```

Comment:
Hm, we can remove the inner pragma for now. It’ll beef up the file size if you use `data` and don’t want `is` though.